neonvm/controllers/metrics.go

```diff
@@ -24,8 +32,32 @@ func MakeReconcilerMetrics() ReconcilerMetrics {
 			},
 			[]string{"controller"},
 		),
+		vmCreationToRunnerCreationTime: prometheus.NewHistogram(
+			prometheus.HistogramOpts{
+				Name:    "vm_creation_to_runner_creation_duration_seconds",
+				Help:    "Time duration from VirtualMachine.CreationTimestamp to runner Pod.CreationTimestamp",
+				Buckets: buckets,
+			},
+		),
+		runnerCreationToVMRunningTime: prometheus.NewHistogram(
+			prometheus.HistogramOpts{
+				Name:    "runner_creation_to_vm_running_duration_seconds",
```

Comment:
nit: _maybe_ prepend `vm_` to the start of this name, so that these have a common prefix?
wdyt?

Reply:
fixed in c7654bb64d275c857d62acb8332b58c76a203f34